src/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/Analytic.hpp

```diff
@@ -0,0 +1,81 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <memory>
+#include <pup.h>
+#include <string>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/Gauges.hpp"
+#include "Options/Options.hpp"
+#include "PointwiseFunctions/InitialDataUtilities/InitialData.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+class DataVector;
+namespace gsl {
+template <class T>
+class not_null;
+}  // namespace gsl
+template <size_t Dim>
+class Mesh;
+/// \endcond
+
+namespace GeneralizedHarmonic::gauges {
+/*!
+ * \brief Imposes the analytic gauge condition,
+ * \f$H_a=\Gamma_a^{\mathrm{analytic}}\f$.
+ *
+ * \warning Assumes \f$\partial_t \Gamma_a=0\f$ i.e. the solution is static.
+ */
+class Analytic final : public GaugeCondition {
```

Comment:
How do you feel about naming this `AnalyticChristoffel`? Just `Analytic` doesn't really give any insight into what the gauge is because technically $H_a = 0, H_a = 2 \Gamma_a^{\mathrm{Analytic}}$ are all analytic as well, but aren't what we *mean* when we say analytic.